addons/website_sale_stock/static/src/xml/website_sale_stock_product_availability.xml

```diff
@@ -8,6 +8,20 @@
                 <t t-if='has_out_of_stock_message' t-out='out_of_stock_message'/>
                 <t t-elif="!allow_out_of_stock_order">Out of Stock</t>
             </div>
+            <t t-if="free_qty lte 0 and !cart_qty">
+                <a href="#" class="show_notification_email_form">Be notified when back in stock</a>
+                <div class="email_notification_form d-none">
+                    <form action="/shop/stock_email_notification" name="email_notification_form">
+                        <div class="input-group w-75">
+                            <input name="email" class="form-control" type="text" placeholder="john.doe@example.com"/>
+                            <input type="hidden" name="product_id" t-att-value="product_id"/>
+                            <div class="input-group-append">
+                                <a href="#" role="button" class="btn btn-secondary a-submit">Get Notified</a>
+                            </div>
+                        </div>
+                    </form>
+                </div>
+            </t>
```

Comment:
This has the exact same condition as the div (`id="out_of_stock_message"`) above, can't it be part of it?